massa-node/base_config/config.toml

```diff
@@ -39,6 +39,8 @@
     enable_broadcast = false
     # deferred credits delta (in milliseconds)
     deferred_credits_delta = 7776000000 # ~ 3 months (90×24×60×60×1000) in milliseconds
+    # max datastore keys query (also used by GRPC API)
+    #max_datastore_keys_query = 500
```

Comment:
there should be 2 distinct config parameters:
- max datastore keys queries: limits the number of simultaneous key queries in the api
- max datastore keys limit : limits the number of keys returned by a single query